modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -158,25 +152,38 @@ deployment.apps/logging-eventrouter created
 
 . Validate that the Event Router installed in the `openshift-logging` project:
 +
+.. View the new Event Router Pod:
++
 [source,terminal]
 ----
-$ oc get pods --selector  component=eventrouter -o name
+$ oc get pods --selector  component=eventrouter -o name -n openshift-logging
 ----
 +
 .Example output
 [source,terminal]
 ----
 pod/cluster-logging-eventrouter-d649f97c8-qvv8r
 ----
+
+.. View the events collected by the Event Router:
 +
 [source,terminal]
 ----
-$ oc logs cluster-logging-eventrouter-d649f97c8-qvv8r
+$ oc logs <cluster-logging-eventrouter-pod> -n openshift-logging 
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc logs cluster-logging-eventrouter-d649f97c8-qvv8r -n openshift-logging 
 ----
 +
 .Example output
 [source,terminal]
 ----
-{"verb":"ADDED","event":{"metadata":{"name":"kibana.1612e7cc3325689f","namespace":"openshift-logging","selfLink":"/api/v1/namespaces/openshift-logging/events/kibana.1612e7cc3325689f","uid":"2e20b62e-ebe8-4eb5-9c25-1d7b54fd294a","resourceVersion":"805355","creationTimestamp":"2020-05-27T14:08:02Z"},"involvedObject":{"kind":"Deployment","namespace":"openshift-logging","name":"kibana","uid":"1970b521-bd9e-49e8-ab7b-9f6403a2ba2f","apiVersion":"apps/v1","resourceVersion":"805353"},"reason":"ScalingReplicaSet","message":"Scaled up replica set kibana-6674598965 to 1","source":{"component":"deployment-controller"},"firstTimestamp":"2020-05-27T14:08:02Z","lastTimestamp":"2020-05-27T14:08:02Z","count":1,"type":"Normal"}}
+{"verb":"ADDED","event":{"metadata":{"name":"openshift-service-catalog-controller-manager-remover.1632d931e88fcd8f","namespace":"openshift-service-catalog-removed","selfLink":"/api/v1/namespaces/openshift-service-catalog-removed/events/openshift-service-catalog-controller-manager-remover.1632d931e88fcd8f","uid":"787d7b26-3d2f-4017-b0b0-420db4ae62c0","resourceVersion":"21399","creationTimestamp":"2020-09-08T15:40:26Z"},"involvedObject":{"kind":"Job","namespace":"openshift-service-catalog-removed","name":"openshift-service-catalog-controller-manager-remover","uid":"fac9f479-4ad5-4a57-8adc-cb25d3d9cf8f","apiVersion":"batch/v1","resourceVersion":"21280"},"reason":"Completed","message":"Job completed","source":{"component":"job-controller"},"firstTimestamp":"2020-09-08T15:40:26Z","lastTimestamp":"2020-09-08T15:40:26Z","count":1,"type":"Normal"}}
 ----
++
+You can also use Kibana to view events by creating an index pattern using the Elasticsearch `infra` index.
```

Comment:
Noticed this is the only one where it's `infra` index instead of `.infra` index. Dunno if intentional.